### ISSUE TITLE
Cannot Turn Off 2nd Display for LG V60

### ISSUE THREAD

**Issue by MSFF11** [Genymobile/scrcpy#3716]
- [ x] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [ x] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Windows 11 Version 10.0.22621 Build 22621
 - scrcpy version: 1.25
 - installation method: Windows release
 - device model: LG V60 LM-V600TM
 - Android version: 12

**Describe the bug**

I have a LG V60 with dual screens and mirror them via usb cable to my PC. I want to turn off both displays with `MOD`+`o`, but it can only turn off the main phone display and not the 2nd one.

I've tried clicking on the 2nd display window and press `MOD`+`o`, but it still only turns off the main display. I've closed the main display window and kept the 2nd one opened and still the same issue.

Please paste terminal output in a code block.
```scrcpy 1.25 <https://github.com/Genymobile/scrcpy>
D:\scrcpy\scrcpy-server: 1 file pushed, 0 skipped. 2.2 MB/s (42151 bytes in 0.018s)
[server] INFO: Device: LGE LM-V600 (Android 12)
INFO: Renderer: direct3d
INFO: Initial texture: 1080x2456
[server] INFO: Device screen turned off
```

I apologize if I'm not clear enough or more info is needed, this is my first time using Github. I figure this is a pretty rare use case. Please let me know if anything else is needed

**Comment by rom1v** [comment]
I just attempted a fix on branch [`physical_displays`](https://github.com/Genymobile/scrcpy/commits/physical_displays).

Please test :)

Here are a binary to replace in your 1.25 release:

- [`scrcpy-server`](https://tmp.rom1v.com/scrcpy/3716/1/scrcpy-server) <sub>`SHA-256: f06d0a1f64704af79768efbf704c1128b1b8afd9e17765e2385aaee1faa32558`</sub>

> I apologize if I'm not clear enough or more info is needed, this is my first time using Github.

Your issue is probably in the top 10% of bug reports in quality :wink:

**Comment by MSFF11** [comment]
Looks like I accidentally closed this.

I've tested it with the new binary. It allows me to turn off both displays simultaneously when pressing 'MOD'+'o'. I'm not sure if there's someone who wants to do each screen independently, but currently it turns them both off.

Sadly, touching the screen doesn't function anymore. Whenever I click on either screen, I get this:

```
scrcpy 1.25 <https://github.com/Genymobile/scrcpy>
D:\scrcpy\scrcpy-server: 1 file pushed, 0 skipped. 89.4 MB/s (78916 bytes in 0.001s)
[server] INFO: Device: LGE LM-V600 (Android 12)
INFO: Renderer: direct3d
INFO: Initial texture: 1080x2456
[server] INFO: Device screen turned off
[server] INFO: Device screen turned on
[server] WARN: Unknown event type: -1
[server] WARN: Unknown event type: -1
[server] WARN: Unknown event type: -1
[server] WARN: Unknown event type: -1
```

Sometimes clicking on the home screen registered it as a long press and wanted me to edit my Home Screen. Most of the time, whenever I click, nothing happens on screen. 

I'm also using the default launcher for this device, which is probably [LG V60 ThinQ Launcher]

I've tried using it without mirroring the 2nd display and the same result.

**Comment by rom1v** [comment]
> Sadly, touching the screen doesn't function anymore.

Oh, yes, that's because I based this change on `dev` branch instead of `master`.

> I've tested it with the new binary. It allows me to turn off both displays simultaneously when pressing 'MOD'+'o'. I'm not sure if there's someone who wants to do each screen independently, but currently it turns them both off.

OK, so it works as expected. I'll merge that. Thank you for your tests. :+1:

**Comment by rom1v** [comment]
Merged into `dev`: f2dee20a207b9906574d9b3d62a0ab2585dade20
